unicode/Makefile

```diff
@@ -1,6 +1,9 @@
-Scheme=../bin/scheme
+Scheme=../bin/zuo .. run
 
-doit: unicode-char-cases.ss unicode-charinfo.ss
+doit: update unicode-char-cases.ss unicode-charinfo.ss
+
+update:
+	./get-UNIDATA
```

Comment:
I'm guessing that the intent of this is to run `update` (to completion) before the other two targets?  Listing them this way allows all three to run in parallel (or any arbitrary order).

Maybe
```
doit:
    ${MAKE} update
    ${MAKE} unicode-char-cases.ss unicode-charinfo.ss
```
(with tabs substituted appropriately)

...although that will still allow parallel execution if someone explicitly names the targets together on the command line.  (I'm not too concerned about that scenario... I say that falls under "play stupid games, win stupid prizes")